.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: 🗣 Subproject discussions 
```

Comment:
I would remove lines 3-5 as the link goes to QualityOnDemand discussions and Discussion feature is not activated for this repository